t/plugin/limit-count3.t

```diff
@@ -224,3 +224,64 @@ passed
     }
 --- response_body
 [200,200]
+
+
+
+=== TEST 7: set another route with the same conf
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/2',
+                ngx.HTTP_PUT,
+                [[{
+                    "uri": "/hello1",
+                    "plugins": {
+                        "limit-count": {
+                            "count": 2,
+                            "time_window": 60
+                        }
```

Comment:
the conf of `limit-count` different form route 1

```
    "limit-count": {
        "count": 2,
        "time_window": 61
    }
```

is the same conf means the value of attr such as `count` or `time_window`?

Reply:
> is the same conf mean the value of attr such as count or time_window all equal to another limit-count in other routes?

Yes.

Reply:
but the `time_window = 60` in TEST 7 is different form the `time_window = 61` TEST 5?

It seems that TEST 5&7 should test the same conf and not affect each other.

Reply:
Yes, you are right! Fixed.